source/_integrations/spotify.markdown

```diff
@@ -72,29 +72,45 @@ client_secret:
 <div class='note'>
 
   If you receive an `INVALID_CLIENT: Invalid redirect URI` error while trying to
-  authenticate with your Spotify account, make sure to check the Redirect URI in
+  authenticate with your Spotify account, check the Redirect URI in
   the address bar after adding the new integration. Compare this value with the
   Redirect URI defined in the Spotify Developer Portal.
 
 </div>
 
 ## Using multiple Spotify accounts
 
-The integration support multiple Spotify accounts at once. You don't need to
+This integration supports multiple Spotify accounts at once. You don't need to
 create another Spotify application in the Spotify Developer Portal and no
 modification to the `configuration.yaml` file is needed. Multiple Spotify
 accounts can be linked to a _single_ Spotify application.
 
-To add an additional Spotify account to Home Assistant, repeat _only_ the steps
-in the [Activating the Spotify integration](#activating-the-spotify-integration)
-section. However, make sure you go to the Spotify website and log out before
-you do so.
+To add an additional Spotify account to Home Assistant, go to the Spotify website and log out, then repeat _only_ the steps
+in the [Activating the Spotify integration](#activating-the-spotify-integration) section. 
 
-## URI Links for Playlists
+## Playing Spotify playlists
 
 You can send playlists to Spotify using the `"media_content_type": "playlist"`, which is part of the
-[media_player.play_media](/integrations/media_player/#service-media_playerplay_media) service.
-The `media_content_id` can be obtained from the Spotify desktop-app by clicking on the more options ("...") next to the album art picture, selecting "share" and then "Copy Spotify URI" or "Copy Playlist Link" (also available in the Spotify phone and web app).
+[media_player.play_media](/integrations/media_player/#service-media_playerplay_media) service, for example:
+```yaml
+# as part of a script
+  alias: Jazz Guitar
+  sequence:
+  - service: media_player.play_media
+    data:
+      entity_id: media_player.spotify
+      media_content_id: 'https://open.spotify.com/playlist/5xddIVAtLrZKtt4YGLM1SQ?si=YcvRqaKNTxOi043Qn4LYkg'
+      media_content_type: playlist
+```
+
+The `media_content_id` value can be obtained from the Spotify desktop app by clicking on the more options ("...") next to the album art picture, selecting "Share" and then "Copy Spotify URI" or "Copy Playlist Link" (also available in the Spotify phone and web app).
+
+<div class='note'>
+
+  There is currently a problem targeting Google Nest Home media player devices (in `entity_id`). 
+  These do not currently play Spotify playlists, but other controls do work.
+
+</div>
```

Comment:
This is not related to this integration. Please remove this note.

Reply:
Where could we put this instead? One of the biggest problems I have with HA documentation is that I try something; it doesn't work and then I find out on the forums that it's been a problem for a long time. This Spotify documentation is where I, as a user, would expect to find current limitations/problems with playing Spotify playlists.
Where would you suggest would be better for users looking to play playlists?

Reply:
@frenck - I have removed this for now, but please consider how to better support users with "just in time" location of this sort of information.

Reply:
The note you are trying to add is a limitation for cast devices. Probably the `cast` integration is a better place for such a note. This note wasn't related to any of this. This integration cannot change any of that behavior either.